Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
-gemspec
-gem 'rubocop'
+gem 'coco'
```

Comment:
Oops...that got left in there because I was trying to see if using another coverage tool helped with the coverage problems I was seeing. It didn't.